lld/MachO/Options.td

```diff
@@ -396,6 +396,9 @@ def dead_strip : Flag<["-"], "dead_strip">,
 def interposable : Flag<["-"], "interposable">,
     HelpText<"Indirects access to all exported symbols in an image">,
     Group<grp_opts>;
+def read_threads : Joined<["--"], "read-threads=">,
+    HelpText<"Number of threads to use to proactively page in files for faster disk i/o. 20 would be a typical value, use 0 to disable this feature.">,
```

Comment:
Where does the 20 comes from? Is it dependent in your own computer? The number is very close to `2 * number of cores` of a typical laptop. 20 might not be the right number for other machines (different number of cores, different hard drives).

But as I wrote in another comment above. The current code does not really use this number as a number of threads, so the help text is lying.

```suggestion
    HelpText<"Number of threads to eagerly preload input files content into memory. Use 0 to disable this feature. Default is disabled.">,
```